src/bundle/Resources/config/services/ui_config_mappers.yml

```diff
@@ -0,0 +1,24 @@
+parameters:
+    ezplatform.field_type.ezrichtext.custom_tags.translation_domain: 'custom_tags'
```

Comment:
I'm not sure if this is ever going to change. Also we are hardcoding translation domain names everywhere.

Reply:
> I'm not sure if this is ever going to change. Also we are hardcoding translation domain names everywhere.

I'm using it twice, so I prefer to parametrize it. IMHO parametrizing something that is used multiple times is better than hardcoding, even if no-one would ever need to override it.